lib/modules/apostrophe-pages/index.js

```diff
@@ -99,7 +99,8 @@ module.exports = {
                  if (idea.location && idea.location.coordinates[0] && idea.location.coordinates[1]) {
                    let lat = idea.location.coordinates[0];
                    let lng = idea.location.coordinates[1];
-                   idea.locationMapImage = `https://maps.googleapis.com/maps/api/staticmap?center=${lat},${lng}&zoom=15&size=380x214&maptype=roadmap&markers=icon:${appUrl}/img/idea/flag-red.png|${lat},${lng}&key=${googleMapsApiKey}`;
+                   let flag = req.data.global.mapImageFlag ? 'flag-' + req.data.global.mapImageFlag : 'flag-red';
+                   idea.locationMapImage = `https://maps.googleapis.com/maps/api/staticmap?center=${lat},${lng}&zoom=15&size=380x214&maptype=roadmap&markers=icon:${appUrl}/img/idea/${flag}.png|${lat},${lng}&key=${googleMapsApiKey}`;
```

Comment:
Is this the only place where we want to set the mapImageFlag? 
What about the flags in the map-widget?

Reply:
The flags in the map-widget depend on a lot more factors, for instance the theme of the idea and the status. The map image does not depend on any logic, so it should be sufficient to select this through the global settings.